### PR TITLE
[doc]rocksdb compaction flush

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
add rocksdb compaction flush and update toc

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+update configs storage:rocksdb_column_family_options = { disable_auto_compactions = true }
+```
+
+### 打开自动 Compaction
+
+在 console 输入如下命令打开自动 compaction。
+
+```bash
+update configs storage:rocksdb_column_family_options = { disable_auto_compactions = false }
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md]: 大写命令 UPDATE CONFIGS

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+## 手动触发 compaction 和 flush
+
+手动触发 compaction 和 flush 需要通过 HTTP 请求，命令如下。
+
+```bash
+curl ${ws_ip}:${ws_http_port}/admin?space=${spaceName}&${op}
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md]: curl  "${ws_ip}:${ws_http_port}/admin?space=${spaceName}&${op}"

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+- `ws_ip` 为 HTTP 服务的 IP，可以在 `storage.conf` 配置文件中找到。
+- `ws_http_port` 为 storage 的 HTTP 服务端口。
+- `op` 为相关的 admin 操作，当前只支持 `compact` 和 `flush`。
+
+例如：
+
+```bash
+curl http://127.0.0.1:50005/admin?space=test&op=compact
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md]: curl "http://127.0.0.1:50005/admin?space=test&op=compact"
curl "http://127.0.0.1:50005/admin?space=test&op=flush"

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# RocksDB Compaction & Flush
+
+Nebula supports auto and manual (through HTTP request) RocksDB compaction or flush in storage service.
+
+## Auto RocksDB Compaction
+
+### Turn off auto Compaction
+
+Input the following command in console to turn off the auto compaction.
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/rocksdb-compaction-flush.md]: **turn off **

**auto compaction**

**Comment by jude-zhu** [comment]
genkins go

**Pull request merged**